.htmltest.external.yml

```diff
@@ -0,0 +1,16 @@
+DirectoryPath: public/docs
+IgnoreDirectoryMissingTrailingSlash: true
```

Comment:
@celestehorgan , 
Did you find a large number of "trailing slash" link errors (`IgnoreDirectoryMissingTrailingSlash`=true)?
What is the `CheckLinks` setting?

Reply:
Setting this to `false` _exlcudes_ missing trailing slashes in the final report. 

`IgnoreDirectoryMissingTrailingSlash: true` = 24 errors
 `IgnoreDirectoryMissingTrailingSlash: false` = 19 errors

Not significant either way.